www/tutorial/index.html

```diff
@@ -24,6 +24,6 @@
 
     <script src="dist/inspec_tutorial.js"></script>
 
-    <my-app class="inspec-tutorial">Loading...</my-app>
+    <div inspec-tutorial class="inspec-tutorial">Loading...</div>
```

Comment:
fyi: the `my-app` element rips the middleman build part into pieces; that's why I changed it into an attribute